Dockerfile

```diff
@@ -1,11 +1,12 @@
-FROM golang:1.14 AS build
+FROM registry.redhat.io/ubi8/ubi AS build
```

Comment:
what about `registry.redhat.io/ubi8/go-toolset` ?

Reply:
This seems to cover a very specific usecase:
```
$ podman run -it registry.redhat.io/ubi8/go-toolset
This is a S2I go-toolset base image:
To use it, install S2I: https://github.com/openshift/source-to-image
Sample invocation:
s2i build -e IMPORT_URL='github.com/cpuguy83/go-md2man' git://github.com/cpuguy83/go-md2man rhel8/go-toolset go-sample-app
```
When trying to use it, `RUN yum install make` fails, apparently because by default commands do not run as root in this image. Probably simpler to stick with the base ubi image.
```
STEP 3: RUN yum -y install make
2020-12-10 08:18:07,810 [ERROR] yum:1:MainThread @logutil.py:194 - [Errno 13] Permission denied: '/var/log/rhsm/rhsm.log' - Further logging output will be written to stderr
Not root, Subscription Management repositories not updated

This system is not registered to Red Hat Subscription Management. You can use subscription-manager to register.

Error: This command has to be run with superuser privileges (under the root user on most systems).
```

Reply:
Yes go tool set is only to build go application, you are not able to install any dep on top of it.